apps/public-docsite-v9/src/DocsComponents/FluentDocsPage.stories.tsx

```diff
@@ -133,30 +144,108 @@ const getNativeElementsList = (elements: SBEnumType['value']): JSX.Element => {
   );
 };
 
-const RenderArgsTable = ({ hideArgsTable, primaryStory }: { primaryStory: PrimaryStory; hideArgsTable: boolean }) => {
+const slotRegex = /as\?:\s*"([^"]+)"/;
+
+function withSlotEnhancer(story: PreparedStory) {
```

Comment:
lest add docs that this mutates original reference

```suggestion
/**
* NOTE: this function mutates original story argTypes including all story subcomponents if they are present
*/
function withSlotEnhancer(story: PreparedStory) {
```

Reply:
Thank you, updated accordingly.